packages/react-native-otlp/src/utils.ts

```diff
@@ -0,0 +1,17 @@
+const createFalsePromise = () => {
```

Comment:
I'm not sure if these are needed, could use Promise.reject/resolve instead: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/reject